### DISCUSSION TITLE
ayuda

### DISCUSSION THREAD

**Discussion by bombardeo38** [Jyers/Marlin#1506] [Q&A] [answered]
necesito ayuda me gusta este friware pwro al utilizar la cama de cristal me golpea en las grapa podrias hacer versiones para el tamaño 220X220x250 con la nivelacion de la cama tanto por bl como la que hace con la boquilla 3x3 gracias de ante mano

**Comment by Cal-LF** [comment, marked as answer]
It is not practical to make versions for different bed sizes. Compiling your own firmware is the way to adjust the firmware to meet your personal requirements. 
You can also position the bed clips so they are not struck by the nozzle.
